app/views/admin/users/actions/_export.html.erb

```diff
@@ -0,0 +1,19 @@
+<div id="export-data">
+  <div class="flex flex-col gap-4">
+    <p>
+      You can export a user's data here. Currently we only support exporting a user's posts and comments.
+    </p>
+    <p>
+      You have the option of exporting to your admin account to send to the user, or exporting to the user directly.
```

Comment:
**_Quick suggestion:_** after speaking with some folks recently about whether or not "Admin" should be capitalized, it seems like it should be, as it is a proper noun, so I suggest that we capitalize it here (and everywhere else that it pops up!)!
```suggestion
      You have the option of exporting to your Admin account to send to the user, or exporting to the user directly.
```